### DISCUSSION TITLE
Code-N-Collab is participating in HacktoberFest

### DISCUSSION THREAD

**Discussion by atharmohammad** [atharmohammad/Code-N-Collab#175] [Announcements]
want to start contributing to opensource :sunglasses:   ? don't know where to start :disappointed:  ? New to opensource :thinking:  ?
you can start contributing in code-n-collab :100:  , this hacktober fest we would be helping begginer's with the difficulty they might be facing while contributing :grinning:  
1). setup development environment(both front-end and backend) following ReadME
-- Stuck somewhere ask here in discussions
2). Pick any issue you like 
-- confused or stuck = ask
3). Finally , generate a PR

**Comment by anishwar-007** [comment]
i am having a difficulty in setting up the local enviroment variables 
     REACT_APP_SIGNALLING_URL1 = wss://y-webrtc-signaling-eu.herokuapp.com
     REACT_APP_SIGNALLING_URL2 = wss://y-webrtc-signaling-us.herokuapp.com
     REACT_APP_BASE_URL = 'http://localhost:8080/'
just pasting this in the cmd prompt is not working

**Reply by atharmohammad** [reply]
Create a ```.env``` file and paste these variables there and put this file in the root directory

**Reply by anishwar-007** [reply]
then i can just start the local server with npm start?

**Reply by atharmohammad** [reply]
Yes! but make sure you have already started the backend services at http://localhost:8080/'

**Reply by anishwar-007** [reply]
It worked! thanks.
